client/app/state/series/sagas/getSerie.js

```diff
@@ -1,10 +1,15 @@
-import { call, put, takeLatest } from 'redux-saga/effects';
+import { /*call, */put, takeLatest } from 'redux-saga/effects';
 import { getSerieRequest, getSerieDone, getSerieError } from '../actions/getSerie';
-import { getSerieById } from 'services';
+//import { getSerieById } from 'services';
+import fakeSeries from '../../../../serie.json';
 
-function *getSerieAction(action) {
+
+function *getSerieAction(/*action*/) {
     try {
-        const serie= yield call(getSerieById, action.payload);
+        //const serie= yield call(getSerieById, action.payload);
+        const serie = {
+            data: fakeSeries
+        } 
```

Comment:
Remove all comments

Reply:
I will remove the coments when the API exist, at the moment is the way i am using in mobile development, i can make a localhost request but it only works in ipv4 and i have to change continuously when y chane my network